project/HmrcBuild.scala

```diff
@@ -37,8 +37,9 @@ object Dependencies {
     "uk.gov.hmrc" %% "crypto" % "2.0.0",
     "uk.gov.hmrc" %% "http-verbs" % "3.0.0",
     "uk.gov.hmrc" %% "play-auditing" % "0.2.0",
-    "uk.gov.hmrc" %% "play-filters" % "2.0.0",
+    "uk.gov.hmrc" %% "play-filters" % "3.0.0",
     "uk.gov.hmrc" %% "play-graphite" % "2.0.0",
+    "uk.gov.hmrc" %% "play-config" % "2.0.1",
```

Comment:
play config should not be here, it's the job of the services to bring this in if they need it